### DISCUSSION TITLE
一分钟读论文：《ChatGPT：在线考试诚信的终结？》

### DISCUSSION THREAD

**Discussion by giscus** [unbug/unbug.github.io#22] [Announcements]
# 一分钟读论文：《ChatGPT：在线考试诚信的终结？》

新西兰奥克兰梅西大学数学与计算科学学院的论文《ChatGPT: The End of Online Exam Integrity?》评估了 ChatGPT 执行高级认知任务和生成与人类生成的文本的能力。发现：ChatGPT 能够展现批判性思维能力。

https://unbug.github.io/ChatGPT-The-End-of-Online-Exam-Integrity/

**Comment by unbug** [comment]
很多大学生都不具备批判性思维，但 ChatGPT 就玩溜了。
>[大学生批判性思维现状及成因研究](http://qks.cqu.edu.cn/html/cqdxskcn/2016/3/20160328.htm)制定了批判性思维倾向问卷(修改版)量表满分为70分，总得分≤42分表明批判性思维弱，＞42分≤49分表明批判性思维处于中下水平，＞49分≤56分表明批判性思维处于中上水平，＞56分表明批判性思维强。
发现受试者中有40.11%的学生低于42分，68.03%的受试者批判性思维都在49分及以下，大于56分的学生只占10.07%，说明受试者中大部分学生的批判性思维相对较弱。
